plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/DefaultQueryBuilder.java

```diff
@@ -176,6 +176,54 @@ public PreparedQuery prepareDeleteQuery(
         return new PreparedQuery(sql, accumulator.build());
     }
 
+    @Override
+    public PreparedQuery prepareUpdateQuery(
+            JdbcClient client,
+            ConnectorSession session,
+            Connection connection,
+            JdbcNamedRelationHandle baseRelation,
+            TupleDomain<ColumnHandle> tupleDomain,
+            Optional<ParameterizedExpression> additionalPredicate,
+            List<JdbcAssignmentItem> assignments)
+    {
+        ImmutableList.Builder<QueryParameter> accumulator = ImmutableList.builder();
+
+        String sql = "UPDATE " + getRelation(client, baseRelation.getRemoteTableName()) + " SET ";
+
+        sql += assignments.stream().map(entry -> {
+                    JdbcColumnHandle columnHandle = entry.getColumn();
+
+                    String bindExpression = getWriteFunction(
+                            client,
+                            session,
+                            connection,
+                            columnHandle.getJdbcTypeHandle(),
+                            columnHandle.getColumnType())
+                            .getBindExpression();
+
+                    accumulator.add(
+                            new QueryParameter(
+                                    columnHandle.getJdbcTypeHandle(),
+                                    columnHandle.getColumnType(),
+                                    entry.getQueryParameter().getValue()));
+                    return client.quoted(columnHandle.getColumnName()) + " = " + bindExpression;
+                }
+        ).collect(joining(", "));
```

Comment:
nit: indentation is odd. Move `)` to previous line.

Reply:
refactored